genesyscloud/telephony_providers_edges_extension_pool/genesyscloud_telephony_providers_edges_extension_pool_proxy.go

```diff
@@ -0,0 +1,125 @@
+package telephony_providers_edges_extension_pool
```

Comment:
My comments for edge groups apply here too

Reply:
addressed